Jenkinsfile

```diff
@@ -31,39 +28,34 @@ node {
-              passwordVariable: 'BINTRAY_APIKEY', usernameVariable: 'BINTRAY_USERNAME')]) {
-                chartVersion = "${prNumber}.${env.BUILD_NUMBER}"
-                pega_chartName = "pega-${chartVersion}.tgz"
-                addons_chartName = "addons-${chartVersion}.tgz"
-                DELETE_STATUS_CODE = sh(script: "curl -X DELETE -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/index.yaml --write-out '%{http_code}'", returnStdout: true).trim()
-                PEGA_STATUS_CODE = sh(script: "curl -T ${pega_chartName} -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/ --write-out '%{http_code}'", returnStdout: true).trim()
-                ADDONS_STATUS_CODE = sh(script: "curl -T ${addons_chartName} -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/ --write-out '%{http_code}' ", returnStdout: true).trim()
-                UPDATE_STATUS_CODE = sh(script: "curl -T index.yaml -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/ --write-out '%{http_code}'", returnStdout: true).trim()
-                PUBLISH_STATUS_CODE = sh(script: "curl -X POST -u${BINTRAY_USERNAME}:${BINTRAY_APIKEY} https://api.bintray.com/content/pegasystems/helm-test-automation/helm-test-automation/${bintrayPackageVersion}/publish --write-out '%{http_code}'", returnStdout: true).trim()
-                echo "DELETE_STATUS_CODE-- ${DELETE_STATUS_CODE}"
-                echo "PEGA_STATUS_CODE-- ${PEGA_STATUS_CODE}"
-                echo "ADDONS_STATUS_CODE-- ${ADDONS_STATUS_CODE}"
-                echo "UPDATE_STATUS_CODE-- ${UPDATE_STATUS_CODE}"
-                echo "PUBLISH_STATUS_CODE-- ${PUBLISH_STATUS_CODE}"
+            // Publish helm charts to test-automation GitHub Pages
+            withCredentials([usernamePassword(credentialsId: "helmautomation",
+              passwordVariable: 'AUTOMATION_APIKEY', usernameVariable: 'AUTOMATION_USERNAME')]) {
 
-                if ( "${DELETE_STATUS_CODE}" != "${curlSuccessStatus}"+"200" || "${PEGA_STATUS_CODE}" != "${curlSuccessStatus}"+"201" || "${ADDONS_STATUS_CODE}" != "${curlSuccessStatus}"+"201"
-                      || "${UPDATE_STATUS_CODE}" != "${curlSuccessStatus}"+"201" || "${PUBLISH_STATUS_CODE}" != '{"files":3}'+"200" ) {
-                    currentBuild.result = 'FAILURE'
-                    pullRequest.comment("Unable to publish helm charts to bintray repository. Please retry")
-                    error "This pipeline stops here! Unable to perform helm charts publish to bintray repository."
+                sh "git clone https://pegaautomationuser:${AUTOMATION_APIKEY}@github.com/pegaautomationuser/helmcharts.git --branch=gh-pages gh-pages"
```

Comment:
Can you read username through `AUTOMATION_USERNAME` variable?

Reply:
Addressed in latest commit